released/packages/coq-simple-io/coq-simple-io.1.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "Li-yao Xia <lysxia@gmail.com>"
+authors: "Li-yao Xia"
+homepage: "https://github.com/Lysxia/coq-simple-io"
+bug-reports: "https://github.com/Lysxia/coq-simple-io/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/Lysxia/coq-simple-io.git"
+build: [
+  [make "build"]
+  [make "test"] {with-test}
+]
+install: [make "install"]
+remove: [make "uninstall"]
+depends: [
+  "ocaml"
+  "coq"
```

Comment:
Please specify an interval that this package is compatible with. It can be either a range of specific versions, `{ >= "8.9" & "8.10~"}` or an upwards open interval, `{>= "8.9"}`.